package/gluon-config-mode-outdoor/luasrc/lib/gluon/config-mode/wizard/0250-outdoor.lua

```diff
@@ -0,0 +1,28 @@
+
+return function(form, uci)
+	local outdoor_mode = uci:get_bool('gluon', 'wireless', 'outdoor')
+
+	if not outdoor_mode then
+		-- only shown when enabled. enabled through preset or web-wifi-config.
+		return
+	end
```

Comment:
A checkbox you can only disable, but not enable seems weird. Either always show the checkbox (if the wifi supports 5GHz), or only show a note like the autoupdater.

Reply:
The idea is that the checkbox only shows up if the outdoor-mode is enabled. It is meant as a little reminder that the outdoor-mode is enabled (for example if the autodetection system enabled the outdoor-mode) at the moment and you can disable it directly in the wizard.
You can also enable the outdoor-mode on non-auto-enabled devices via the wifi tab in the advanced settings.

imho this keeps the wizard clean from outdoor-mode related messages that you do not need. (e.g. devices that are not made for outdoor usage)

Reply:
It is very weird to allow disabling an option through the wizard, but being unable to revert this change without the advanced settings (and it would even be possible to create a firmware that has the wizard checkbox, but not the advanced settings page, leaving only SSH or a factory reset to revert it).

Reply:
Maybe we can show the checkbox on auto-enabled devices everytime and do not show it (when outdoor-mode is disabled) on all other 5GHz devices.
Then you could enable and disable the outdoor-mode on all auto-enabled devices directly in the wizard and for all other 5GHz devices the advanced settings are the way to go. This would keep the wizard clean for the majority of all devices and still offers the outdoor-mode-settings.
@NeoRaider @mweinelt Does this sound like a good solution for you?

Reply:
Still a bit weird, but it sounds acceptable to me.

Reply:
Addressed in df0ce8f4ea9357dd223d97a393911bedcaee8852